src/cgame/cg_predict.cpp

```diff
@@ -398,7 +398,7 @@ static void CG_TouchTriggerPrediction()
 			continue;
 		}
 
-		if ( ent->eType == entityType_t::ET_TELEPORTER )
+		if ( ent->eType == entityType_t::ET_TELEPORTER && ent->eFlags & ~EF_NODRAW )
```

Comment:
Checking for any flag except EF_NODRAW? Surely that can't be right. Maybe you meant `!( ent->eFlags & EF_NODRAW )`

Reply:
`EF_NODRAW` is the flag set when the teleporter is disabled, so I think this is correct?

Reply:
The condition as written ignores whether EF_NODRAW is set and checks whether any other flag is set.

Reply:
Turns out I cannot read code properly. I've fixed this in the latest force-push.